### PR TITLE
Update VRF Heat Pump Cooling/Heating COP documentation

### LINKED ISSUE #10084: VRF Heat Pump Cooling/Heating COP documentation update
Issue overview
--------------
The documentation for the VRF Heat Pump Cooling/Heating COP output variables is out of date.

### Details
```
1.32.1.2.7 VRF Heat Pump Cooling COP[]
This is the operating cooling coefficient of performance (COP) for the heat pump. This value is
calculated using the ratio of VRF Heat Pump Total Cooling Rate and Variable Refrigerant Flow Heat
Pump Cooling Electric Consumption Rate output variables. Crankcase heater (usually 0 in cooling mode),
evaporative condenser water pump, and defrost (usually 0 in cooling mode) consumption rate output
variables are included in this calculation. This value is specific to outdoor unit performance in cooling
mode, is calculated for each HVAC system time step being simulated, and the results are averaged for the
time step being reported.

1.32.1.2.8 VRF Heat Pump Heating COP[]
This is the operating heating coefficient of performance (COP) for the heat pump. This value is
calculated using the ratio of VRF Heat Pump Total Heating Rate and Variable Refrigerant Flow Heat
Pump Heating Electric Consumption Rate output variables. Crankcase heater, evaporative condenser
water pump (usually 0 in heating mode), and defrost consumption rate output variables are included in
this calculation. This value is specific to outdoor unit performance in heating mode, is calculated for each
HVAC system time step being simulated, and the results are averaged for the time step being reported.
```

The denominator variables were changed in previous E+ versions:

v7.2.0 to v8.0.0
Variable Refrigerant Flow Heat Pump Cooling Electric Consumption Rate -> VRF Heat Pump Cooling Electric Power
v9.3.0 to v9.4.0
VRF Heat Pump Cooling Electric Power -> VRF Heat Pump Cooling Electricity Rate

v7.2.0 to v8.0.0
Variable Refrigerant Flow Heat Pump Heating Electric Consumption Rate -> VRF Heat Pump Heating Electric Power
v9.3.0 to v9.4.0
VRF Heat Pump Heating Electric Power -> VRF Heat Pump Heating Electricity Rate

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by mdahlhausen**
The denominator variables were changed in previous E+ versions:

v7.2.0 to v8.0.0
Variable Refrigerant Flow Heat Pump Cooling Electric Consumption Rate -> VRF Heat Pump Cooling Electric Power
v9.3.0 to v9.4.0
VRF Heat Pump Cooling Electric Power -> VRF Heat Pump Cooling Electricity Rate

v7.2.0 to v8.0.0
Variable Refrigerant Flow Heat Pump Heating Electric Consumption Rate -> VRF Heat Pump Heating Electric Power
v9.3.0 to v9.4.0
VRF Heat Pump Heating Electric Power -> VRF Heat Pump Heating Electricity Rate

Pull request overview
---------------------
 - Fixes #10084 
 - Corrects variable names in I/O reference.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@mdahlhausen @myoldmopar it has been 29 days since this pull request was last updated.

**Comment by nrel-bot-2b** [comment]
@mdahlhausen it has been 11 days since this pull request was last updated.

**Comment by nrel-bot-2** [comment]
@mdahlhausen it has been 7 days since this pull request was last updated.

**Comment by nrel-bot-2** [comment]
@Myoldmopar @mdahlhausen it has been 7 days since this pull request was last updated.

File: doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex [hunk 1]
```diff
@@ -639,11 +639,11 @@ \subsubsection{Outputs}\label{outputs-039}
...
+This is the operating cooling coefficient of performance (COP) for the heat pump. This value is calculated using the ratio of VRF Heat Pump Total Cooling Rate and VRF Heat Pump Cooling Electricity Rate output variables. Crankcase heater (usually 0 in cooling mode), evaporative condenser water pump, and defrost (usually 0 in cooling mode) consumption rate output variables are included in this calculation. This value is specific to outdoor unit performance in cooling mode, is calculated for each HVAC system time step being simulated, and the results are averaged for the time step being reported.
```
Comment (rraustad) [inline doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex:642]: Hmm, how does this relate to non-electric systems?

    SetupOutputVariable(state, format("VRF Heat Pump Cooling {} Rate", sFuelType),

    // Calculate OperatingHeatingCOP & OperatingCoolingCOP: VRF Heat Pump Operating COP []
    if (state.dataHVACVarRefFlow->CoolingLoad(VRFCond) && CoolingPLR > 0.0) {
        if (this->ElecCoolingPower != 0.0) {
            // this calc should use delivered capacity, not condenser capacity, use VRF(VRFCond).TUCoolingLoad
            this->OperatingCoolingCOP = (this->TotalCoolingCapacity) /
                                        (this->ElecCoolingPower + this->CrankCaseHeaterPower + this->EvapCondPumpElecPower + this->DefrostPower);
        } else {
            this->OperatingCoolingCOP = 0.0;
        }
    }

Comment (Myoldmopar) [reply doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex:642]: It seems this is still an open question.  I'll hold from reviewing this for right now.

Comment (rraustad) [reply doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex:642]: I realize this change only updates the documentation to currently reported text strings. But as I looked at this, this other question popped up. Either use this branch as an update to the correct text string and open another issue to investigate COP with non-electric fuel types, or check into what COP means with non-electric equipment on this branch and update the docs accordingly (or whatever else is needed). I don't use non-electric equipment very often so I don't have a suggestion off the top of my head.

Comment (Myoldmopar) [reply doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex:642]: Given the timing, I think I will just accept this doc update on its own, and we can investigate COP with non-electric later.

**Comment by Myoldmopar** [comment]
OK, merging this doc-only change in, more can be done in a follow-up as needed.  Thanks @mdahlhausen and @rraustad

**Pull request merged**